lib/connection/connection.js

```diff
@@ -140,14 +140,25 @@ function Connection(context)
     });
   };
 
+  /**
+   * @return {Promise<boolean>}
```

Comment:
This is like partially documented. By the name of the function I know what the function implements, but only the return type is different than in other places